contracts/strategies/ltip-hedgey/LTIPHedgeyStrategy.sol

```diff
@@ -0,0 +1,190 @@
+
+    // @notice Initialize the strategy
+    /// @param _poolId ID of the pool
+    /// @param _data The data to be decoded
+    /// @custom:data (address hedgeyContract, address vestingAdmin, bool adminTransferOBO, InitializeParams initializeParams)
+    function initialize(uint256 _poolId, bytes memory _data) external virtual override {
+        (InitializeParamsHedgey memory initializeParamsHedgey) = abi.decode(_data, (InitializeParamsHedgey));
+        __LTIPHedgeyStrategy_init(_poolId, initializeParamsHedgey);
+        emit Initialized(_poolId, _data);
+    }
+
+    /// @notice This initializes the BaseStrategy
+    /// @dev You only need to pass the 'poolId' to initialize the BaseStrategy and the rest is specific to the strategy
+    /// @param _initializeParams The initialize params
+    function __LTIPHedgeyStrategy_init(uint256 _poolId, InitializeParamsHedgey memory _initializeParams) internal {
+        // Initialize the BaseStrategy
+        __LTIPSimpleStrategy_init(_poolId, _initializeParams.initializeParams);
+
+        hedgeyContract = _initializeParams.hedgeyContract;
+        vestingAdmin = _initializeParams.vestingAdmin;
+        adminTransferOBO = _initializeParams.adminTransferOBO;
+    }
+
+    /// ===============================
+    /// ============ Views ============
+    /// ===============================
+
+    /// ===============================
+    /// ======= External/Custom =======
+    /// ===============================
+
+    /// @notice Update the default admin wallet used when creating Hedgey plans
+    /// @param _vestingAdmin The new admin address
+    function setVestingAdmin(address _vestingAdmin) external onlyPoolManager(msg.sender) {
+        vestingAdmin = _vestingAdmin;
+        emit AdminAddressUpdated(vestingAdmin, msg.sender);
+    }
+
+    /// @notice Update the default sdmin wallet used when creating Hedgey plans
+    /// @param _adminTransferOBO Set if the admin is allowed to transfer on behalf of the recipient
+    function setAdminTransferOBO(bool _adminTransferOBO) external onlyPoolManager(msg.sender) {
+        adminTransferOBO = _adminTransferOBO;
+        emit AdminTransferOBOUpdated(_adminTransferOBO, msg.sender);
+    }
+
+    /// ====================================
+    /// ============ Internal ==============
+    /// ====================================
+
+    /// @notice Creates a Hedgey vesting plan for the recipient
+    /// @param recipientId The recipient ID
+    /// @param recipientAddress The recipient address
+    /// @param _token The address of the tokens to vest
+    /// @param _amount The amount to vest
+    function _vestAmount(address recipientId, address recipientAddress, address _token, uint256 _amount)
+        internal
+        virtual
+        override
+    {
+        Recipient memory recipient = _recipients[recipientId];
+
+        IERC20(_token).approve(hedgeyContract, _amount);
+
+        uint256 rate = _amount / recipient.allocationAmount;
```

Comment:
this will always be `1` since in `LTIPSimpleStrategy` it's called like this:
`_vestAmount(recipientId, recipient.recipientAddress, pool.token, recipient.allocationAmount);`

which results in 
```
// _amount == recipient.allocationAmount;
// and therefore
// rate = _amount / recipient.allocationAmount = recipient.allocationAmount  / recipient.allocationAmount  = 1
```

Reply:
Reached out to project team if we want to expose all params and move this input client side or opt for vesting/streaming setup with their respective configs

Reply:
Will be adding the hedgey vesting parameters to the init struct, as we'd otherwise settle for an arbitrary distribution of e.g. 2 weeks. To leave this up to the round operator, exposing the parameters is probably cleanest